trollflow2/launcher.py

```diff
@@ -438,3 +440,61 @@ def sendmail(config, trace):
     pid = Popen([sendmail, "-t", "-oi"], stdin=PIPE)
     pid.communicate(msg.as_bytes())
     pid.terminate()
+
+
+def launch(args_in):
+    """Launch the processing."""
+    args = parse_args(args_in)
```

Comment:
do we need `args_in` here, or does argparse get this by itself?

Reply:
For much simpler tests.